### PR TITLE
Url/markdowns

### PR DESCRIPTION
**Pull request by Morriden**

### CODE REVIEW

File: src/actions/actions.js [hunk 1]
```diff
@@ -14,3 +17,9 @@ export const newMarkdown = title => ({
     body: ''
   }
 });
+
+export const UPDATE_ID = 'UPDATE_ID';
```
Comment (mehtaphysical) [inline src/actions/actions.js:21]: you dont need this anymore

File: src/components/markdown/Editor.jsx [hunk 2]
```diff
@@ -3,17 +3,19 @@ import { useDispatch, useSelector } from '../../hooks/appContext';
 import { getMarkdown } from '../../selectors/markdownSelectors';
 import { updateMarkdown } from '../../actions/actions';
 import styles from './Editor.css';
+import { useParams } from 'react-router-dom';
 
 const Editor = () => {
-  const markdown = useSelector(getMarkdown);
   const dispatch = useDispatch();
+  const { id } = useParams() || 1234;
```
Comment (mehtaphysical) [inline src/components/markdown/Editor.jsx:10]: that works. more idiomatic is:

```js
const { id = 1234 } = useParams()
```

File: src/components/markdown/Preview.jsx [hunk 3]
```diff
@@ -2,10 +2,13 @@ import React from 'react';
 import styles from 'github-markdown-css/github-markdown.css';
 import { useSelector } from '../../hooks/appContext';
 import { getMarkdownPreview } from '../../selectors/markdownSelectors';
+import { useParams } from 'react-router-dom';
 
 const Preview = () => {
 
-  const __html = useSelector(getMarkdownPreview);
+  const { id } = useParams() || 1234;
```
Comment (mehtaphysical) [inline src/components/markdown/Preview.jsx:9]:
```js
const { id = 1234 } = useParams()
```

**Comment by Morriden** [comment]
Comments changed and updated

**Pull request merged**